tests/installation/grub_test.pm

```diff
@@ -80,9 +73,21 @@ sub run() {
         }
         send_key 'ret';
     }
-    if (get_var("XEN")) {
+    elsif (get_var("XEN")) {
         send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5);
     }
+    elsif (check_var('ARCH', 'x86_64') && check_var('DISTRI', 'sle')) {
+        record_soft_failure "bsc#995310 - Gnome fails to start";
```

Comment:
yes, we saw this together with the guys in QAM openQA workshop and it confused so I think reverting is a better choice and still - as I recommended - just make it optional with a variable. I guess it's better to clone some test_suites (or at least one) but keep the defaults one intact, i.e. with splash screen shown.